README.md

```diff
@@ -123,6 +113,11 @@ func main() {
 ## Sealed results
 
 This SDK provides utility methods for decoding [sealed results](https://dev.fingerprint.com/docs/sealed-client-results).
+Install the sealed results dependency as below:
+```shell
+go get github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/v5/sdk/sealed
+```
```

Comment:
I'm not sure if this necessary. For me, running this command just adds `github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/v5` to my `go.mod`

Reply:
Without this command, the example code throws error

```
missing go.sum entry for module providing package github.com/pkg/errors (imported by github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/v5/sdk/sealed); to add:
	go get github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/v5/sdk/sealed@v5.0.1
```